packages/microwave/src/lib/microwave.spec.ts

```diff
@@ -0,0 +1,166 @@
+import { ChangeDetectorRef, Component } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { createObserver } from '../../testing/observer';
+import { finalize } from 'rxjs';
+import { Microwave, watch } from './microwave';
+
+@Component({
+  template: `{{ meal }} is {{ evaluation }}`,
+})
+@Microwave()
+class GreetingsComponent {
+  meal?: string;
+  evaluation = 'meh';
+}
+
+jest.useFakeTimers();
```

Comment:
Why do we need this?

Reply:
Oh right! I added it to flush micro tasks and macro tasks then I switched to a simple `Promise.resolve()` so it's not needed anymore.